phys2bids/interfaces/acq.py

```diff
@@ -42,8 +39,8 @@ def populate_phys_input(filename, chtrig):
         warnings.filterwarnings('ignore', category=DeprecationWarning)
         data = read_file(filename).channels
 
-    freq = [data[chtrig - 1].samples_per_second, ]
-    timeseries = [data[chtrig - 1].time_index, ]
+    freq = [data[0].samples_per_second, ]
+    timeseries = [data[0].time_index, ]
```

Comment:
```suggestion
    frequencies = [x.samples_per_second for x in data]
    max_index = [i for i, j in enumerate(frequencies) if j == max(frequencies)][0]
    freq = [data[max_index].samples_per_second, ]
    interval = 1/freq[0]
    duration = (timeseries[max_index].shape[max_index] + 1) * interval
    timeseries = [np.ogrid[0:duration:interval][:-1], ]  # create time channel
```
This will make your frequency time and time channel independent from any index because you will always get the biggest one